paramiko/pkey.py

```diff
@@ -97,7 +107,7 @@ class PKey:
             "mode": modes.CBC,
         },
         "DES-EDE3-CBC": {
-            "cipher": algorithms.TripleDES,
+            "class": TripleDES,
```

Comment:
The key name shouldn't change here

Reply:
No wonder, didn't even notice that...fat-fingered the keyboard somewhere 🤦